deployment/ansible/roles/cac-tripplanner.app/tasks/main.yml

```diff
@@ -50,6 +50,17 @@
   template: src=nginx-default.j2 dest=/etc/nginx/sites-available/default
   notify: Restart nginx
 
+- name: Touch cron job log file if it does not exist
+  command: touch {{ app_cron_event_feed_log }}
+           creates={{ app_cron_event_feed_log }}
+
+- name: Set cron job log file permissions
+  file: path={{ app_cron_event_feed_log }} owner={{ app_username }} group={{ app_username }} mode=0664
+
+# TODO: Add logic in production to only run on a single webserver
```

Comment:
As I recall, we put a bunch of points on this card.  Would be good to go ahead and do this now.